.browserslistrc

```diff
@@ -4,8 +4,8 @@ not dead
 
 # Browsers with more than 0.1% global usage, excluding old browsers
 > 0.1%
-not chrome < 45
-not ie < 9
+not ie < 11
+not safari < 9

```

Comment:
Safari 5.1 shows high usage, which is probably due to Safari for Windows, which is no longer supported.

Reply:
@ylemkimon this seems reasonable.  I am curious about what you're looking at for usage numbers.  Can you post a link to your source?

Reply:
@kevinbarabash `Can I use` has a browser usage table: https://caniuse.com/usage-table. `browserslist` also uses this via package `caniuse-lite`.

Reply:
I've used caniuse.com for quite some time, but I've never seen that particular chart.  The Chrome for Android number seems a little suspicious.  Regardless, cool chart!